arangod/Aql/AqlValueGroup.h

```diff
@@ -34,14 +34,24 @@ struct Options;
 
 namespace aql {
 
+using AqlValueGroup = std::vector<AqlValue>;
+
+struct HashedAqlValueGroup {
+  size_t hash;
```

Comment:
Shall we have a default initializer for `hash`, e.g. 0?
I know `hash` will be overriden later, and before it is read from, but adding a default value here may be more intuitive (at least for me it is).